### DISCUSSION TITLE
Cannot Locate OsmAnd Support Group Chat

### DISCUSSION THREAD

**Discussion by JackZ295** [osmandapp/OsmAnd#13855] [Q&A] [answered]
Hi! I remember that OsmAnd also had a group chat for support, in addition to this GitHub forum. Would someone mind pointing me in the right direction? Thanks!

**Comment by JackZ295** [comment, marked as answer]
I just realized that the OsmAnd Support Group Chat is on Telegram

**Comment by tommycrock** [comment]
There's also the Google group for users helping users https://groups.google.com/g/osmand / osmand@googlegroups.com
